modules/datadog-agent/default.auto.tfvars

```diff
@@ -12,9 +12,7 @@ repository = "https://helm.datadoghq.com"
 
 chart = "datadog"
 
-# https://github.com/DataDog/helm-charts/releases
-# 2.26.2 released on Nov 19, 2021
-chart_version = "2.26.2"
+chart_version = "2.32.6"
```

Comment:
Also I think we're moving all of these out of default.auto.tfvars now, right ?

Reply:
AFAIK we were keeping them in `default.auto.tfvars`